lib/config/rule-validator.js

```diff
@@ -169,7 +169,15 @@ class RuleValidator {
                 if (validateRule.errors) {
                     throw new Error(`Key "rules": Key "${ruleId}": ${
                         validateRule.errors.map(
-                            error => `\tValue ${JSON.stringify(error.data)} ${error.message}.\n`
+                            error => {
+                                if (error.keyword === "additionalProperties" && error.schema === false) {
```

Comment:
```suggestion
                                if (error.keyword === "additionalProperties" && error.schema === false && typeof error.parentSchema?.properties === "object" && typeof error.params?.additionalProperty === "string") {
```

It might be good to check if all expected data is there so that this doesn't crash in case Ajv doesn't always return everything we're expecting here.

Reply:
Done.